vec/vec-v113/src/main/resources/generate-xjb-from-vec-model.xsl

```diff
@@ -106,10 +101,7 @@
         <xsl:variable name="tempName" select="normalize-space(.)"/>
         <xsl:variable name="name">
             <!-- strip leading '/' -->
-            <xsl:if test="substring($tempName,1,1) != '/'">
-                <xsl:value-of select="translate(substring($tempName,1,1),'abcdefghijklmnopqrstuvwxyz','ABCDEFGHIJKLMNOPQRSTUVWXYZ')"/>
-            </xsl:if>
-            <xsl:value-of select="substring($tempName,2,string-length($tempName))"/>
+            <xsl:if test="substring($tempName,1,1) != '/'"><xsl:value-of select="translate(substring($tempName,1,1),'abcdefghijklmnopqrstuvwxyz','ABCDEFGHIJKLMNOPQRSTUVWXYZ')"/></xsl:if><xsl:value-of select="substring($tempName,2,string-length($tempName))"/>
```

Comment:
Because correct generation beats readability. The content of `<xsl:attribtute>` is added completely to the attribute (including whitespace). The "readable" version in the current develop produces garbage when executed like:

```xml
   <jxb:bindings node="//xs:complexType[@name='DocumentRelatedAssignmentGroup']&#xA;                ">
      <jxb:bindings node=".//xs:element[@name='RelatedDocumentVersion']&#xA;                ">
         <nav:ext-reference schema-type="DocumentVersion"
                            inverse="ref&#xA;                        DocumentRelatedAssignmentGroup"/>
      </jxb:bindings>
      <jxb:bindings node=".//xs:element[@name='RelatedSheetOrChapter']&#xA;                ">
         <nav:ext-reference schema-type="SheetOrChapter"
                            inverse="ref&#xA;                        DocumentRelatedAssignmentGroup"/>
      </jxb:bindings>
   </jxb:bindings>
```